src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+USE jdbc;
+CREATE TABLE books (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Primary key already contains not null constraint in it
```suggestion
    id INT AUTO_INCREMENT PRIMARY KEY,
```